src/simple_cache_client.rs

```diff
@@ -1093,6 +1107,21 @@ impl SimpleCacheClient {
         self.data_client.delete(request).await?.into_inner();
         Ok(())
     }
+
+    fn validate_ttl(&self, policy: &TtlPolicy) -> MomentoResult<Duration> {
+        let ttl = policy.ttl().unwrap_or(self.item_default_ttl);
+        utils::is_ttl_valid(ttl)?;
+        Ok(ttl)
+    }
+
+    fn expand_ttl_ms(&self, policy: TtlPolicy) -> MomentoResult<u64> {
+        let ttl_ms = self
+            .validate_ttl(&policy)?
+            .as_millis()
+            .try_into()
+            .unwrap_or(u64::MAX);
```

Comment:
can you use `i64::MAX as u64` instead just so there's 1 less potential issue with any proxy we may have? It doesn't materially change the effect.